apps/nextjs/next.config.js

```diff
@@ -1,6 +1,11 @@
 import _jiti from "jiti";
 
-const jiti = _jiti(new URL(import.meta.url).pathname);
+let path = new URL(import.meta.url).pathname;
+
+if (process.platform === "win32") {
```

Comment:
can use [`pathToFileURL`](https://nodejs.org/api/url.html#urlpathtofileurlpath) which handles cross-platform paths

Reply:
Thanks for the suggestion. I updated my PR.